app/src/main/java/org/wikipedia/page/PageFragment.kt

```diff
@@ -1429,7 +1427,7 @@ class PageFragment : Fragment(), BackPressedHandler, CommunicationBridge.Communi
 
         override fun onViewEditHistorySelected() {
             title?.run {
-                startActivity(EditHistoryListActivity.newIntent(requireContext(), this))
+                startActivity(EditHistoryListActivity.newIntent(requireContext(), this, model.page?.run { pageProperties.pageId } ?: -1))
```

Comment:
Don't have to use `run {}` for getting variables. Just simply using the null check with express symbols is fine.

```
model.page?.pageProperties?.pageId ?: -1
```